pkg/driver/controller.go

```diff
@@ -412,6 +434,14 @@ func (cs *controller) deleteVolume(ctx context.Context, volumeID string) error {
 				"failed to handle delete volume request for {%s}", volumeID)
 		}
 	}
+	present := isNodePresent(vol.Spec.OwnerNodeID)
```

Comment:
what if the finalizer is already removed? Then we don't need to run this?

Reply:
Right now this basically removes all Finalizers.  I am using the function which is called by node-agent once `lvremove` is done.  

https://github.com/openebs/lvm-localpv/blob/develop/pkg/lvm/volume.go#L217C1-L223C2

`lvmvolume` CR is managed by localpv-lvm. Users may not add any finalizers onto it.

Reply:
Is this documented somewhere that users may not add finalizers?
It seems we just take the hammer there